app/src/test/java/org/easyrecipe/features/search/SearchViewModelTest.kt

```diff
@@ -58,6 +61,8 @@ class SearchViewModelTest {
         time = 75
     ))
 
+    private lateinit var mainViewModel: MainViewModel
```

Comment:
The tests should only test the behavior of a single class and all dependencies should be either a fake or a mock. In the project we should use mocks.